pkg/agent/openflow/network_policy.go

```diff
@@ -1823,13 +1827,55 @@ func newFeatureNetworkPolicy(
 }
 
 func (f *featureNetworkPolicy) initFlows() []binding.Flow {
-	f.egressTables = map[uint8]struct{}{EgressRuleTable.GetID(): {}, EgressDefaultTable.GetID(): {}}
+	f.egressTables = map[uint8]struct{}{EgressDefaultTable.GetID(): {}}
 	if f.enableAntreaPolicy {
 		f.egressTables[AntreaPolicyEgressRuleTable.GetID()] = struct{}{}
 	}
 	var flows []binding.Flow
-	flows = append(flows, f.establishedConnectionFlows()...)
-	flows = append(flows, f.relatedConnectionFlows()...)
-	flows = append(flows, f.ingressClassifierFlows()...)
+	if f.nodeType == config.K8sNode {
+		f.egressTables[EgressRuleTable.GetID()] = struct{}{}
+		flows = append(flows, f.ingressClassifierFlows()...)
+	}
+	flows = append(flows, f.skipPolicyRuleCheckFlows()...)
+	return flows
+}
+
+// skipPolicyRuleCheckFlows installs OpenFlow entries to resubmit the packets in an established or related
+// connections to the metric table in the same stage directly. It also resubmits the packets marked with
+// "CustomReasonReject" to the metric table. So that these packets would skip the check on NetworkPolicy rules.
```

Comment:
Probably "skip NetworkPolicy rules" or "skip the flows to check NetworkPolicy rules"

Reply:
updated